modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-test/src/testIntegration/java/com/liferay/headless/commerce/admin/order/resource/v1_0/test/ChannelResourceTest.java

```diff
@@ -14,15 +14,327 @@
+						"JSONObject/data",
+						"Object/orderByExternalReferenceCodeChannel"))));
+	}
+
+	@Override
+	@Test
+	public void testGraphQLGetOrderIdChannel() throws Exception {
+		Channel channel = testGraphQLGetOrderIdChannel_addChannel();
+
+		Assert.assertTrue(
+			equals(
+				channel,
+				ChannelSerDes.toDTO(
+					JSONUtil.getValueAsString(
+						invokeGraphQLQuery(
+							new GraphQLField(
+								"orderIdChannel",
+								HashMapBuilder.<String, Object>put(
+									"id", _commerceOrder.getCommerceOrderId()
+								).build(),
+								getGraphQLFields())),
+						"JSONObject/data", "Object/orderIdChannel"))));
+	}
+
+	@Override
+	protected String[] getAdditionalAssertFieldNames() {
+		return new String[] {"currencyCode", "name"};
+	}
+
+	@Override
+	protected Channel randomChannel() throws Exception {
+		return new Channel() {
+			{
+				currencyCode = StringUtil.toLowerCase(
+					RandomTestUtil.randomString());
+				externalReferenceCode = StringUtil.toLowerCase(
+					RandomTestUtil.randomString());
+				id = RandomTestUtil.randomLong();
+				name = StringUtil.toLowerCase(RandomTestUtil.randomString());
+				type = CommerceChannelConstants.CHANNEL_TYPE_SITE;
+			}
+		};
+	}
+
+	@Override
+	protected Channel testGetOrderByExternalReferenceCodeChannel_addChannel()
+		throws Exception {
+
+		return _toChannel();
+	}
+
+	@Override
+	protected Channel testGetOrderIdChannel_addChannel() throws Exception {
+		return _toChannel();
+	}
+
+	@Override
+	protected Channel testGetOrderRuleChannelChannel_addChannel()
+		throws Exception {
+
+		return _toChannel();
+	}
+
+	@Override
+	protected Long testGetOrderRuleChannelChannel_getOrderRuleChannelId()
+		throws Exception {
+
+		return _corEntryRel.getCOREntryRelId();
+	}
+
+	@Override
+	protected Channel testGetOrderTypeChannelChannel_addChannel()
+		throws Exception {
+
+		return _toChannel();
+	}
+
+	@Override
+	protected Long testGetOrderTypeChannelChannel_getOrderTypeChannelId()
+		throws Exception {
+
+		return _commerceOrderTypeRel.getCommerceOrderTypeRelId();
+	}
+
+	@Override
+	protected Channel testGraphQLChannel_addChannel() throws Exception {
+		return _toChannel();
+	}
+
+	@Override
+	protected Long testGraphQLGetOrderRuleChannelChannel_getOrderRuleChannelId()
+		throws Exception {
+
+		return _corEntryRel.getCOREntryRelId();
+	}
+
+	@Override
+	protected Long testGraphQLGetOrderTypeChannelChannel_getOrderTypeChannelId()
+		throws Exception {
+
+		return _commerceOrderTypeRel.getCommerceOrderTypeRelId();
+	}
+
+	private Channel _toChannel() throws Exception {
+		return new Channel() {
+			{
+				currencyCode = _commerceChannel.getCommerceCurrencyCode();
+				externalReferenceCode =
+					_commerceChannel.getExternalReferenceCode();
+				id = _commerceChannel.getCommerceChannelId();
+				name = _commerceChannel.getName();
+				type = _commerceChannel.getType();
+			}
+		};
+	}
+
+	@Inject
+	private static AccountEntryLocalService _accountEntryLocalService;
```

Comment:
The variable is static based on this doc: https://help.liferay.com/hc/en-us/articles/360017883412-Injecting-Service-Components-into-Tests